components/org.wso2.carbon.identity.api.server.claim.management/org.wso2.carbon.identity.rest.api.server.claim.management.v1/src/main/java/org/wso2/carbon/identity/rest/api/server/claim/management/v1/core/ServerClaimManagementService.java

```diff
@@ -666,7 +658,12 @@ private LocalClaimResDTO getLocalClaimResDTO(LocalClaim localClaim) {
 
         Map<String, String> claimProperties = new HashMap<>(localClaim.getClaimProperties());
 
-        localClaimResDTO.setDescription(claimProperties.remove(PROP_DESCRIPTION));
+        String description = claimProperties.remove(PROP_DESCRIPTION);
+        if (description != null) {
+            localClaimResDTO.setDescription(description);
+        } else {
+            localClaimResDTO.setDescription(StringUtils.EMPTY);
+        }
```

Comment:
```
localClaimResDTO.setDescription(description);
if (description ==  null) {
    localClaimResDTO.setDescription(StringUtils.EMPTY);
}
```
WDYT? If this is ok we can do the same for the `regex` right?

Reply:
In this case when the description is null the response JSON will not get a description key. That might affect some behaviors because of missing keys (Previously the description key was always their in the response).